lib/kanta_web/live/translations/translation_form_live/components/plural_translation_form/plural_translation_form.html.heex

```diff
@@ -22,51 +22,59 @@
                 <svg class="flex-shrink-0 h-5 w-5 text-slate-500" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" fill="currentColor" aria-hidden="true">
                   <path fill-rule="evenodd" d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z" clip-rule="evenodd"></path>
                 </svg>
-                <a phx-click="navigate" phx-value-to={"/kanta/locales/#{@locale.id}/translations"} class="cursor-pointer ml-4 text-sm font-medium text-slate-400 hover:text-slate-200"><%= @locale.name %></a>
+                <.link patch={"/kanta/locales/#{@locale.id}/translations"} class="cursor-pointer ml-4 text-sm font-medium text-slate-400 hover:text-slate-200"><%= @locale.native_name %></.link>
               </div>
             </li>
             <li>
               <div class="flex items-center">
                 <svg class="flex-shrink-0 h-5 w-5 text-slate-500" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" fill="currentColor" aria-hidden="true">
                   <path fill-rule="evenodd" d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z" clip-rule="evenodd"></path>
                 </svg>
-                <a href="#" aria-current="page" class="cursor-pointer ml-4 text-sm font-medium text-slate-400 hover:text-slate-200"><%= @message.msgid %></a>
+                <a href="#" aria-current="page" class="cursor-pointer ml-4 text-sm font-medium text-slate-400 hover:text-slate-200"><%= String.slice(@message.msgid, 0..30) %></a>
               </div>
             </li>
           </ol>
         </nav>
       </div>
       <div class="mt-2 md:flex md:items-center md:justify-between">
         <div class="flex-1 min-w-0">
-          <h2 class="text-2xl font-bold leading-7 text-primary-dark dark:text-accent-dark sm:text-3xl sm:truncate">
-            Translating '<%= @message.msgid %>' to '<%= @locale.name %>'
+          <h2 class="text-xl font-bold leading-7 text-slate-600 dark:text-content-light sm:text-2xl sm:truncate">
+            Translating <span class="text-primary-dark dark:text-accent-dark underline"><%= String.slice(@message.msgid, 0..30) %></span> to <span class="text-primary-dark dark:text-accent-dark underline"><%= @locale.native_name %></span>
```

Comment:
Is this String.slice/2 to prevent overflow?
I think a css solution should be preferred:

https://tailwindcss.com/docs/text-overflow#truncate